### DISCUSSION TITLE
gRPC

### DISCUSSION THREAD

**Discussion by 0x41337** [oven-sh/bun#4025] [General]
How i can use gRPC with bun ?

**Comment by Jarred-Sumner** [comment]
The blocker right now is HTTP2 support. We haven't implemented it yet.

**Reply by mejiasd3v** [reply]
Hey @Jarred-Sumner any update on this?

**Comment by shishamo** [comment]
hello, and thanks for bun team's work^^

is there any update for the implementation of http2?
it could be nice to use gRpc
